cmd/client-s3-trace_v4.go

```diff
@@ -38,6 +38,14 @@ func newTraceV4() httptracer.HTTPTracer {
 func (t traceV4) Request(req *http.Request) (err error) {
 	origAuth := req.Header.Get("Authorization")
 
+	trace := func() {
+		var reqTrace []byte
+		reqTrace, err = httputil.DumpRequestOut(req, false) // Only display header
```

Comment:
Since this is a closure we should avoid `err` to be aliased with the `return` alias such that it is better to call it 

```
trace := func() {
       reqTrace, rerr := httputil.DumpRequestOut(req, false) // Only display header
       if rerr == nil {
              console.Debug(string(reqTrace))
       }
}
```

Reply:
Also perhaps call it `printTrace()`

Reply:
done